test/unit-disabled/main.tf

```diff
@@ -1,12 +1,58 @@
-module "test" {
+module "test1" {
   source = "../.."
 
+  # add all required arguments
```

Comment:
i think the comment here was misleading, we usually do not add all arguments to `unit-disabled`, just the required ones, testing if `disabled` works.

Reply:
that was just a `paste` mistake. Forgot to remove it

Reply:
Does your comment relate to the entire file i.e. there should only be a single test with the two required arguments? I was inspired by this [test/unit-disabled/main.tf](https://github.com/mineiros-io/terraform-google-service-account-iam/blame/main/test/unit-disabled/main.tf)

Reply:
Thanks for the explanation (and discussion separately). Let us adhere to the template test module structure and comments in the future.